Description: Fix installation directories
 - Do not install rpm to /bin/.
 - Setup default directories so that source RPMs rebuilt on Debian get
   the right directories and also that builds occur in /usr/src/rpm
 - Fix statedir and init.d patch
Author: Michal Čihař <nijel@debian.org>
--- a/Makefile.am
+++ b/Makefile.am
@@ -74,7 +74,7 @@
 pkginclude_HEADERS += build/rpmspec.h
 
 
-rpmbindir = `echo $(bindir) | $(SED) -e s,usr/bin,bin,`
+rpmbindir = $(bindir)
 rpmbin_PROGRAMS = rpm
 
 bin_PROGRAMS =		rpm2cpio rpmbuild
@@ -182,13 +182,13 @@
 
 install-exec-hook:
 	rm -f $(DESTDIR)$(bindir)/rpmquery
-	@LN_S@ ../../bin/rpm $(DESTDIR)$(bindir)/rpmquery
+	@LN_S@ rpm $(DESTDIR)$(bindir)/rpmquery
 	rm -f $(DESTDIR)$(bindir)/rpmverify
-	@LN_S@ ../../bin/rpm $(DESTDIR)$(bindir)/rpmverify
+	@LN_S@ rpm $(DESTDIR)$(bindir)/rpmverify
 	rm -f $(DESTDIR)$(bindir)/rpmsign
-	@LN_S@ ../../bin/rpm $(DESTDIR)$(bindir)/rpmsign
+	@LN_S@ rpm $(DESTDIR)$(bindir)/rpmsign
 	rm -f $(DESTDIR)$(bindir)/rpmdb
-	@LN_S@ ../../bin/rpm $(DESTDIR)$(bindir)/rpmdb
+	@LN_S@ rpm $(DESTDIR)$(bindir)/rpmdb
 
 install-data-local:
 	@case "@host_os@" in \
--- a/configure.ac
+++ b/configure.ac
@@ -994,6 +994,7 @@
 	test -f /etc/lvr-release &&		RPMCANONVENDOR=lvr
 	test -f /etc/yellowdog-release &&	RPMCANONVENDOR=yellowdog
 	test -f /etc/caos-release &&		RPMCANONVENDOR=caos
+	test -f /etc/debian_version &&		RPMCANONVENDOR=rpm
 	;;
 esac
 RPMCANONOS="$host_os_noversion"
--- a/doc/manual/builddependencies
+++ b/doc/manual/builddependencies
@@ -29,7 +29,7 @@
 \verbatim
 	rpm -U /mnt/redhat/comps/powertools/6.2/i386/InDependence-1.0-3.i386.rpm
 	rpm -i /mnt/redhat/comps/dist/6.2/SRPMS/gnorpm-0.9-11.src.rpm
-	cd /usr/src/redhat/SPECS
+	cd /usr/src/rpm/SPECS
 	dep -detail rpmbuild -ba gnorpm.spec >& xxx
 	...
 	(the build will take longer since both dep and strace are pigs)
--- a/macros.in
+++ b/macros.in
@@ -974,7 +974,7 @@
 %_datadir		%{_prefix}/share
 %_sysconfdir		/etc
 %_sharedstatedir	%{_prefix}/com
-%_localstatedir		%{_prefix}/var
+%_localstatedir		@localstatedir@
 %_lib			lib
 %_libdir		%{_exec_prefix}/%{_lib}
 %_includedir		%{_prefix}/include
--- a/platform.in
+++ b/platform.in
@@ -33,7 +33,7 @@
 %_oldincludedir		@oldincludedir@
 %_infodir		@infodir@
 %_mandir		@mandir@
-%_initddir		%{_sysconfdir}/rc.d/init.d
+%_initddir		%{_sysconfdir}/init.d
 # Deprecated misspelling, present for backwards compatibility.
 %_initrddir		%{_initddir}
 
